### PR TITLE
CSS aspect-ratio: add to FF89 relnotes and remove from exper. features

### LINKED ISSUE #9349: Issue with "Firefox 89 for developers": Missing mention of `aspect-ratio` being shipped
MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/89

#### What information was incorrect, unhelpful, or incomplete?

`aspect-ratio` shipped with Firefox 89 _([ref](https://bugzilla.mozilla.org/show_bug.cgi?id=1672073))_, but the notes for developers have no mention of this.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/mozilla/firefox/releases/89`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/89
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/firefox/releases/89/index.html
* Last commit: https://github.com/mdn/content/commit/93667e79c4a3ae558f0102707d3d09972f1b24f9
* Document last modified: 2021-07-14T14:36:44.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
This CSS property was added in FF89 but is not mentioned in the release notes:
- Added release note
- Removed from Experimental features
- Verified BCD is correct
- Removed dev-doc-needed - perhaps prematurely

Fixes #9349

What is missing from a normal docs release is the checkbox item "Docs are adequate".
IMO they are not adequate - I have done minimal "fixes" but I am hoping perhaps someone else proper job or suggest how this might be done. 

How this works docs are old and out of date. All the essential ideas are there but it firefox specific and it is not clear what ended up happening from those docs. It all needs to be updated with the current  state of play.

@rachelandrew is the expert, and already wrote a blog on this that is linked.

Specifically:
- [aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) 
  - is documented and cross links to useful additional documentation
  - the [example](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#examples_of_values_for_aspect-ratio) is simplistic. It would be useful to pull in a brief explanation of what you need to do for this to be useful - ie when would you set it or not set it. There is an implication that you need to set the box width and height attributes in HTML in other docs but that is not clear.
  - The section [mapping_width_and_height_to_aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#mapping_width_and_height_to_aspect-ratio) seems to imply that you don't have to set this aspect-ratio on most browsers but it is worth spelling that out. It may be that someone more expert would infer that.
  - In summary, it would be worth some of the info from the linked doc coming into this doc.
 - [Mapping the width and height attributes of media container elements to their aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/Media/images/aspect_ratio_mapping) - this is supposed to provide the answers to why `aspect-ratio` is important. 
  - It was written early in the spec
  - It refers to properties like `intrinsicsize` which do not appear to have got into spec - but may have made it in as [contain-intrinsic-size](https://developer.mozilla.org/en-US/docs/Web/CSS/contain-intrinsic-size)
  - There is a statement that this only applies to img in FF - https://bugzilla.mozilla.org/show_bug.cgi?id=1583980 - . FF states full support so it is probably not true anymore. 
  - All I have done for this is removed a little FF versioning and linked to `aspect-ratio`.

### CODE REVIEW

**Comment by rachelandrew** [comment]
So the history of this is that we initially had a page describing the mapping functionality. Browsers were using this internally to pick up the height and width of images, to reserve the correct space for it. What was later released was the CSS property available for authors to use. So, I think the `aspect-ratio` page needs examples of actually using the `aspect-ratio` property. I'd be happy to provide those, if you want to raise an issue and assign it to me I can do that. :)

File: files/en-us/web/media/images/aspect_ratio_mapping/index.html [hunk 1]
```diff
@@ -39,7 +38,9 @@ <h2 id="A_new_way_of_sizing_images_before_loading_completes">A new way of sizing
 
 <p>Mozilla then <a href="https://github.com/WICG/intrinsicsize-attribute/issues/16">brought the idea up in the WICG community group</a> and discussed it further until representatives from Chrome were onboard with the idea.</p>
 
-<p>Due to this work, browsers are working on adding a new mechanism for sizing images before the actual image is loaded. Firefox has added an internal <code>aspect-ratio</code> property (in version 69 onwards) that applies to replaced elements, and other related elements that accept <code>width</code> and <code>height</code> attributes. This appears in the browser's internal UA stylesheet, similar to the following:</p>
+<p>Due to this work, browsers are working on adding a new mechanism for sizing images before the actual image is loaded.
```
Comment (rachelandrew) [inline files/en-us/web/media/images/aspect_ratio_mapping/index.html:41]:
```suggestion
<p>Due to this work, browsers have implemented a mechanism for sizing images before the actual image is loaded.
```

**Review by rachelandrew — APPROVED** [review]
I think this is fine - with a little change as the mapping is well supported now.

**Comment by hamishwillee** [comment]
> So the history of this is that we initially had a page describing the mapping functionality. Browsers were using this internally to pick up the height and width of images, to reserve the correct space for it. What was later released was the CSS property available for authors to use. So, I think the aspect-ratio page needs examples of actually using the aspect-ratio property. I'd be happy to provide those, if you want to raise an issue and assign it to me I can do that. :)

Thanks @rachelandrew - issue created and assigned to you: https://github.com/mdn/content/issues/9543

I do think that the [mapping page](https://developer.mozilla.org/en-US/docs/Web/Media/images/aspect_ratio_mapping) could also use some "love". Specifically:
- as a user  I probably want the information in the summary first - start setting the img height/width so that the browser knows your preferred aspect ratio OR you can override this using `aspect-ratio` ( I think if you override this you don't need to set the height/width?)
- It is written in a manner that feels `img` specific and I think aspect-ratio is a container setting.

Am I wrong/missing something?

**Pull request merged**